### ISSUE TITLE
[BUG] Can't start Thermos: java.lang.NoClassDefFoundError: joptsimple/OptionSpec

### ISSUE THREAD

**Issue by Earl0fPudding** [Thorfusion/Thermos#1]
**Server Log:** 
Nothing generated.

**FML Log(s):**
Nothing generated.

**Explanation of issue:**
Tried the latest release, even compiled from master. While starting the server, the following exception appears and the program exits immediately:
```
Exception in thread "main" java.lang.NoClassDefFoundError: joptsimple/OptionSpec
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at cpw.mods.fml.relauncher.ServerLaunchWrapper.run(ServerLaunchWrapper.java:25)
	at cpw.mods.fml.relauncher.ServerLaunchWrapper.main(ServerLaunchWrapper.java:12)
Caused by: java.lang.ClassNotFoundException: joptsimple.OptionSpec
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 4 more
```
No files, logs or anything else generated.

**How to recreate this issue:**

1. Get Debian 11 and OpenJDK 1.8.0_292
2. Download the latest release or build from master.
3. Rename the Thermos-1.7.10-1614.25-libraries.zip to libraries.zip and extract it
4. run the Thermos-1.7.10-1614.Thorfusion-server.jar (even tried parameters like `-XX:+UseG1GC -XX:+UseFastAccessorMethods -XX:+OptimizeStringConcat -XX:MetaspaceSize=1024m -XX:MaxMetaspaceSize=2048m -XX:+AggressiveOpts -XX:MaxGCPauseMillis=10 -XX:+UseStringDeduplication -Xms4G -Xmx6G -XX:hashCode=5 -Dfile.encoding=UTF-8`
5. program exits without creating any folders and just writes the exception as seen above

**Modpack Name:** *(Only if public)*
None yet.
**Mods Installed:**
None.
**Plugins Installed:**
None.
**Warmroast Report:** *(Optional)*
As seen above. Original Thermos from CyberdyneCC works without problems with the same steps.

**Thermos Version:**

**Forge Version:**

**Comment by maggi373** [comment]
try the libraries from r60. I know that our server isnt starting either anymore but i havent checked why yet. Checking now

**Comment by Earl0fPudding** [comment]
Wow, thanks for the quick reply! I just tested my build from master with the libraries.zip from r60 and it works.
It would be nice if you could fix it, so that the actual compiled libraries are working.

**Comment by maggi373** [comment]
hmm, i wasnt actual aware that the compiled libraries was not working, just got that part fixed few days ago and put it into r61

**Comment by Earl0fPudding** [comment]
Hmm okay, please look into it. Let me know if you can reproduce this issue or if you need more information from me, thanks!

Btw I got the following Errors during startup:
```
2022-02-27 23:03:50,695 ERROR Unable to locate appender TerminalConsole for logger com.mojang
2022-02-27 23:03:50,696 ERROR Unable to locate appender TerminalConsole for logger 
2022-02-27 23:03:50,696 ERROR Unable to locate appender TerminalConsole for logger net.minecraft
```
Otherwise server starts up normally with libs from r60.

**Comment by maggi373** [comment]
i have found the issue, the package bundle thingy isnt doing what its supposed to do.

**Comment by maggi373** [comment]
i was able to reproduce it btw

**Comment by Earl0fPudding** [comment]
Okay, great news so far! Hope you can fix it.

**Comment by maggi373** [comment]
ok, have found the problem. I removed the scripts that other people are using since, i dont use them and stuff change and i dont bother to fix them when i dont use them. I might make a proper one now. So the fix is that you need to unzip all the jar files inside the packageBundle zip file into the libraries folder

**Comment by maggi373** [comment]
due to the status and i doubt any of the libraries will change, you only need to do that once

**Comment by maggi373** [comment]
got this from make.sh in cyberdyne/thermos/travii
```bash
cd build/distributions/
unzip Thermos*bundle*zip
rm -r bin/unknown
mv bin libraries
zip -r libraries.zip libraries
cp libraries.zip $HOME/release
```

**Comment by maggi373** [comment]
forget about all that, i did an woopsie, it was too easy fix anyway its late here and i im going to bed. Gonna look more on it tommorrow.

**Comment by maggi373** [comment]
so i've done some research and im going to depricate this fork. I will move to https://github.com/Rus-Minecraft/Thermos and port my and other fixes over to it. This fork branch is so borked and ive been trying to get it working properly and the my and other's fixes are minimal so should be easy to port over.

**Comment by Earl0fPudding** [comment]
Thanks for your work! Please let me know when there are any news.

**Comment by Earl0fPudding** [comment]
Hi there, 
Is this really fixed? It doesn't generate a zip file in the build/distributions folder anymore.
Where is now the official library zip file to use?

**Comment by maggi373** [comment]
its technically fixed as the libraries files that you should use is in r60 release. I have yet to begun the switch to the other repo

**Comment by maggi373** [comment]
been busy on making my own maven n stuff in the background before i can do the switch

**Comment by maggi373** [comment]
# #

> Hmm okay, please look into it. Let me know if you can reproduce this issue or if you need more information from me, thanks!
> 
> Btw I got the following Errors during startup:
> 
> ```
> 2022-02-27 23:03:50,695 ERROR Unable to locate appender TerminalConsole for logger com.mojang
> 2022-02-27 23:03:50,696 ERROR Unable to locate appender TerminalConsole for logger 
> 2022-02-27 23:03:50,696 ERROR Unable to locate appender TerminalConsole for logger net.minecraft
> ```
> 
> Otherwise server starts up normally with libs from r60.

those errors are to be expected. its the log4j mitigation. i still have it in, because that the patch might not work on all systems. its a rare case but it does not hurt to have it.

**Comment by maggi373** [comment]
fixed by 5f7c46f0981a00628ede2c7d14a4e3c39d75d217
